server/controllers/events/events_controller.go

```diff
@@ -89,6 +89,11 @@ type VCSEventsController struct {
 	AzureDevopsRequestValidator     AzureDevopsRequestValidator
 }
 
+//go:generate pegomock generate -m --use-experimental-model-gen --package mocks -o mocks/mock_vcs_post_handler.go VCSPostHandler
+type VCSPostHandler interface {
+	Post(w http.ResponseWriter, r *http.Request)
+}
+
```

Comment:
We should prefer to keep the interfaces at the reader where we can, no need for one other package to take a dependency on this package just because of this.